hitl/run.sh

```diff
@@ -1,36 +1,38 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
 # Title:
 #   Booster hardware-in-the-loop (HITL) test script.
 
 # Enable shell operating mode flags.
 set -eux
 
+BOOSTER="${BOOSTER:-booster-hitl}"
+MQTT_BROKER="${MQTT_BROKER:-mqtt.ber.quartiq.de}"
+# When only one booster is connected, we can use the discovery prefix for all activities.
+PREFIX="${PREFIX:-dt/sinara/booster/+}"
+
 function finish {
     # Always disable the mains supply to Booster using the MQTT-enabled power switch upon
     # completion. Leaving booster enabled can be annoying because of the cooling fans.
-    mosquitto_pub -h mqtt.ber.quartiq.de -t cmnd/tasmota_4F64D0/POWER -m OFF
+    mosquitto_pub -h "${MQTT_BROKER}" -t cmnd/tasmota_4F64D0/POWER -m OFF
 }
 trap finish EXIT
 
-# When only one booster is connected, we can use the discovery prefix for all activities.
-PREFIX=dt/sinara/booster/+
-
 # Set up python for testing
-python3 -m venv --system-site-packages vpy
```

Comment:
Unrelated change. Either provide a rationale or remove.

IIRC on the runner this isn't removed automatically.

Reply:
Locally it failed on the second run since `vpy` already exists.

Reply:
Works fine here.

Reply:
Okay, tested again, working fine now as well without this.